runtime-config-linux.md

```diff
@@ -152,12 +160,38 @@ For example, to run a new process in an existing container without updating limi
 
 #### Disable out-of-memory killer
 
+`disableOOMKiller` contains a flag (`true` or `false`) that enables or disables
+the Out of Memory killer for a cgroup. If enabled (`false`), tasks that attempt
+to consume more memory than they are allowed are immediately killed by the OOM killer.
+The OOM killer is enabled by default in every cgroup using the `memory` subsystem.
+To disable it, specify a value of `true`.
+For more information, see [the memory cgroup man page](https://www.kernel.org/doc/Documentation/cgroups/memory.txt).
+
+###### Example
+
 ```json
     "disableOOMKiller": false
 ```
 
 #### Memory
 
+`memory` represents the cgroup subsystem `memory` and it's used to set limits on memory use of the container.
```

Comment:
And since all the fields inside are optional, I'd like an explicit lead-in like:

> **`memory`** (object, optional) represents…